scss/components/_actor-sheet.scss

```diff
@@ -1,9 +1,15 @@
+@import "colors";
+
 /* Minimum size of actor sheet */
 .actor-sheet {
     min-height: 310px;
     min-width: 500px;
 }
 
+::-webkit-scrollbar-track {
```

Comment:
This doesn't seem to be on the standard track so probably not a great idea to use it as we're making some effort to be usable in Safari, Chrome, and Firefox. Now, if this is just to add some missing support on Chrome and Safari then it's probably ok.

Reply:
At least on Macos Chrome and Safari like to draw scrollbar-track shadow next to the sidebar even when scrolling is not needed and it bothers me visually, so I decided to hide it. Firefox doesn't show it.